src/syncState.js

```diff
@@ -77,7 +77,7 @@ export function createMessageListener({ channel, dispatch, allowed }) {
           dispatch(receiveIniteState(stampedAction.payload));
         }
         return;
-      } else if (allowed(stampedAction.type)) {
+      } else if (allowed(stampedAction.type, stampedAction)) {
```

Comment:
passing type and action are a bit redundant, I think it should only take the action. But it will be a breaking change in this case.

Reply:
Yep, that were my thoughts as well. Would you consider a breaking change for this?

Reply:
I just pushed a change which passes the whole action.

Reply:
I'd like to have a breaking change on this since it will give more flexibility. Please update the code to pass the entire action only. Thank you.

Reply:
If you don't mind, please update the Readme as well. Thank you.